src/Compilers/VisualBasic/Test/Semantic/Compilation/CompilationAPITests.vb

```diff
@@ -1536,12 +1536,6 @@ End Class
 
             Dim ars = arc.ReplaceSyntaxTree(tc, ts)
             Assert.True(arc.ReferenceManagerEquals(ars))
-
-            Dim ar3 = arc.ReplaceSyntaxTree(tc, ta)
-            Assert.True(arc.ReferenceManagerEquals(ar3))
-
-            Dim as1 = ars.ReplaceSyntaxTree(tr, ts)
-            Assert.False(ars.ReferenceManagerEquals(as1))
```

Comment:
These were removed from the C# version of the test when we fixed ```ReplaceSyntaxTree``` to throw when adding a tree that already existed in the ```Compilation```.